spec/lib/psulib_traject/processors/summary_holdings_spec.rb

```diff
@@ -0,0 +1,47 @@
+# frozen_string_literal: true
+
+RSpec.describe PsulibTraject::Processors::SummaryHoldings do
+  describe '::call' do
+    subject(:summary_holdings) { described_class.call(record: record) }
+
+    context 'when there are no summary holdings' do
+      let(:record) { MarcBot.build(:record) }
+
+      it { is_expected.to be_empty }
+    end
+
+    context 'when there is a summary holdings unit' do
+      let(:record) { MarcBot.build(:single_summary_holdings) }
+
+      it 'returns one Unit object' do
+        holdings = summary_holdings.first
+        expect(holdings).to be_a(PsulibTraject::PeriodicalHoldings)
+        expect(holdings.library).to eq('UP-SPECCOL')
+        expect(holdings.location).to eq('HC-SERIALS')
+        expect(holdings.summary).to contain_exactly('v.12(1956/57)-v.40:1-7/8(1984)')
+        expect(holdings.supplement).to contain_exactly('Supplemental')
+        expect(holdings.index).to contain_exactly('Index')
+      end
+    end
+
+    context 'when multiple summary holdings are present' do
+      let(:record) { MarcBot.build(:multiple_summary_holdings) }
+
+      it 'returns two Unit objects' do
+        first_holdings = summary_holdings[0]
+        expect(first_holdings).to be_a(PsulibTraject::PeriodicalHoldings)
+        expect(first_holdings.library).to eq('UP-ANNEX')
+        expect(first_holdings.location).to eq('CATO-2')
+        expect(first_holdings.call_number).to eq('HX1 .M3')
```

Comment:
Can we also check if summary, supplement and index is empty here? 🤔 

```suggestion
        expect(first_holdings.call_number).to eq('HX1 .M3')
        expect(first_holdings.summary).to be_empty
        expect(first_holdings.supplement).to be_empty
        expect(first_holdings.index).to be_empty
```
```

Reply:
yep yep. all done. but, @nicktiberi needs some changes to the data structure, so there will be further changes.

Reply:
ok, I approved but we can hold off on merging 👍🏻  thanks!